src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
No worries, it's a quick search-and-replace, and I'd love for it to be consistent with the rest :-)